sematic/resolvers/worker.py

```diff
@@ -254,7 +256,16 @@ def wrap_main_with_logging():
     prefix = log_prefix(args.run_id, JobType.driver if args.resolve else JobType.worker)
     path = _create_log_file_path("worker.log")
 
-    with ingested_logs(path, prefix):
+    if not log_ingestion_enabled():
+        with ingested_logs(path, prefix):
+            print(
```

Comment:
I believe you need to do `sys.sdtout.flush()` after this print to be sure it actually gets written to stdout. Better safe than sorry.